pkg/component/observability/logging/vali/vali.go

```diff
@@ -62,8 +61,10 @@ const (
 	valiMountPathConfig           = "/etc/vali"
 	valiMountPathInitScript       = "/"
 
-	valitailName            = "gardener-valitail"
-	valitailClusterRoleName = "gardener.cloud:logging:valitail"
+	valitailName                 = "gardener-valitail"
+	openTelemetryCollectorName   = "gardener-opentelemetry-collector"
```

Comment:
Looks like this `const` is not being used anywhere in the code, is it a leftover and shall we still keep it ?

`const`(s) with the same value are defined within:

- `pkg/component/observability/opentelemetry/collector/collector.go`
- `pkg/component/observability/logging/vali/constants/constants.go`

as well, which I believe makes this one redundant. WDYT ?

Same for __L67__, `openTelemetryClusterRoleName` seems to be unused.